### ISSUE TITLE
G15 - link to Trace Center Photosensitive Epilepsy Analysis Tool (PEAT) doesn't work

### ISSUE THREAD

**Issue by scottaohara** [w3c/wcag#1600]
Created PR #1599 to close this issue.

**Comment by alastc** [comment]
Thank you, merged so closing this.
